### ISSUE TITLE
SSL Read error: Connection reset by peer

### ISSUE THREAD

**Issue by ghost** [grote/Transportr#530]
![tmp_6060-img_20181009_1026092011133184](https://user-images.githubusercontent.com/38763911/46656312-46077d00-cb9d-11e8-940b-c132f0b05062.JPG)

**Comment by grote** [comment]
@Violinchen is this still happening with the latest 2.0.4 version?

**Comment by Violinchen** [comment]
file:///home/violine/Downloads/Screenshot Transportr.png
Today there was an error when I chose not to travel by high-speed train. I cannot see more because scrolling is deactivated in transportr.

**Comment by Violinchen** [comment]
Today there was an error when I chose not to travel by high-speed train. I cannot see more because scrolling is deactivated in transportr.
![screenshot transportr](https://user-images.githubusercontent.com/38763911/49939037-ef3f5c00-fedb-11e8-8b53-1694f7e12b59.png)
I tried to change start or end and got some routes. The same when I switched again to every transport means. Then I tried again without high-speed trains and got this error.
